lib/rules/no-curly-component-invocation.js

```diff
@@ -47,14 +49,47 @@ const BUILT_INS = new Set([
 
 const ALWAYS_CURLY = new Set(['yield']);
 
+function toAngleAttrValue(attr) {
+  const node = attr.value;
+  if (node.type === 'SubExpression') {
+    if (node.path.original === 'concat') {
```

Comment:
We can't assume that `node.path` is a `PathExpression` here. We need to check that before accessing `.original`:

```suggestion
    if (node.path.type === 'PathExpression' && node.path.original === 'concat') {
```